stringfuzz/transformers/multiply.py

```diff
@@ -0,0 +1,38 @@
+'''
+Multiplying every integer literal by n and repeating
+every character in a string literal n times for some n
+'''
+
+import random
+import string
+
+from stringfuzz.ast import StringLitNode, IntLitNode
+from stringfuzz.ast_walker import ASTWalker
+from stringfuzz.parser import parse
+
+__all__ = [
+    'multiply',
+]
+
+MAX_FACTOR = 10
+
+class MultiplyTransformer(ASTWalker):
+    def __init__(self, ast, n=random.randint(2,MAX_FACTOR)):
+        super(MultiplyTransformer, self).__init__(ast)
+        self.factor = n
+
+    @property
+    def ast(self):
+        return self._ASTWalker__ast
+
+    def exit_literal(self, literal):
+        if isinstance(literal, StringLitNode):
+            literal.value = literal.value * self.factor
```

Comment:
Ifaz's Repeat<sub>n</sub> is a little different: it repeats every *character* in the string, not the whole string. So, "abc" under Repeat<sub>3</sub> would become "aaabbbccc", not "abcabcabc". Were you aiming to replicate Ifaz's transformation here, or to do a similar but not the same transformation?

Reply:
Good catch! I'll fix it now.

I named it multiply instead of repeat because we aren't really repeating when it comes to integer literals.